go.mod

```diff
@@ -256,7 +256,9 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101500.0-rc.3
+//replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101500.0-rc.3
+
+replace github.com/ethereum/go-ethereum => github.com/yuwen01/op-geth v1.0.6-operatorfee
```

Comment:
needs to be replaced when https://github.com/ethereum-optimism/op-geth/pull/388 is merged